### ISSUE TITLE
+오차역전파

### ISSUE THREAD

**Issue by yuseungwoo** [yuseungwoo/pytorch#6]
import torch 
import torch.nn as nn
import torch.optim as optim

torch.manual_seed(70)   

class Network(nn.Module):
    def __init__(self,i_size,h_size,o_size):
        super(Network,self).__init__()
        self.ih=nn.Linear(i_size,h_size)
        self.ho=nn.Linear(h_size,o_size)
        self.sig=nn.Sigmoid()
    
    def forward(self,x):
        z1=self.ih(x)
        a1=self.sig(z1)
        z2=self.ho(a1)
        a2=self.sig(z2)
        
        return a2

x = torch.Tensor([[0,0],[0,1],[1,0],[1,1]])
t = torch.Tensor([0,1,1,0])

net=Network(2,2,1)

loss_function=nn.BCELoss()

optimizer=optim.SGD(params=net.parameters(), lr=0.5)

STEP = 10000

for step in range(STEP):
    
        net.zero_grad()
        y=net(x)
        loss = loss_function(y.squeeze(), t)
        loss.backward()
        optimizer.step()
        if step % 1000 == 0:
            print("{}:\t {:.4f}".format(step,loss.item()))

pred = net(x).ge(0.5)
print(pred.view(-1))
